rotkehlchen/data/all_assets.json

```diff
@@ -7078,6 +7078,13 @@
         "symbol": "KSM",
         "type": "own chain"
     },
+    "DOT": {
+        "coingecko": "polkadot",
+        "name": "Polkadot",
+        "started": 1590500178,
+        "symbol": "DOT",
+        "type": "own chain"
+    },
```

Comment:
We don't use this file anymore. It's here for compatibility reasons. This change should be reverted and the asset added to one update in https://github.com/rotki/assets I'll add this to an issue

Reply:
Okay, I removed it.